src/Atc.Cosmos/DependencyInjection/CosmosBuilder{T}.cs

```diff
@@ -0,0 +1,28 @@
+using Atc.Cosmos.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Atc.Cosmos.DependencyInjection
+{
+    public class CosmosBuilder<T> : CosmosBuilder, ICosmosBuilder<T>
+        where T : class, ICosmosResource
+    {
+        public CosmosBuilder(
+            IServiceCollection services)
+            : base(services)
+        {
+        }
+
+        public ICosmosBuilder<T> WithChangeFeedProcessor<TProcessor>()
+            where TProcessor : class, IChangeFeedProcessor<T>
+        {
+            Services.TryAddSingleton<ICosmosContainerInitializer, LeasesContainerInitializer>();
```

Comment:
This probably needs to be added even if another ICosmosInitializer has been added.

Reply:
Will fix